src/Middleware/CanAtLeastMiddleware.php

```diff
@@ -14,9 +14,9 @@ class CanAtLeastMiddleware
      * @param  string $permissions
      * @return mixed
      */
-    public function handle($request, Closure $next, $permissions)
+    public function handle($request, Closure $next, ...$permissions)
```

Comment:
Whoops, I missed this one. Does variadic really needed here? I think permissions will always be an array is set as `...`?